engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -833,3 +833,17 @@ INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_vers
 
 -- Fix OS category for Guest OS 'Other PV Virtio-SCSI (64-bit)'
 UPDATE `cloud`.`guest_os` SET category_id = 7 WHERE id = 275 AND display_name = 'Other PV Virtio-SCSI (64-bit)';
+
+-- TODO : Move to 4.16
+ALTER TABLE `cloud`.`user_vm` ADD COLUMN `user_vm_type` varchar(255) DEFAULT "UserVM" COMMENT 'Defines the type of UserVM';
+
+UPDATE `cloud`.`vm_template` set deploy_as_is = 1 where id = 8;
```

Comment:
Add a comment to explain why this is done.

Reply:
Explore - For existing env which has gone through an upgrade, we want to handle it in the java upgrade path by checking the registered template and setting the deploy_as_is to true.